pkg/controller/direct/utils/projectref.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package logging
+package utils
```

Comment:
this may be a bit if bikeshedding but could we rename this pkg to `references` ?

Reply:
Done!

Reply:
I share the same bikeshedding as @acpana 🤣